src/lib/packet.h

```diff
@@ -565,18 +572,29 @@ typedef enum {
 
 #define PGP_SA_DEFAULT_CIPHER PGP_SA_CAST5
 
+#define PGP_SA_DEFAULT_CIPHER_MODE PGP_CIPHER_MODE_CFB
+
 void pgp_calc_mdc_hash(
   const uint8_t *, const size_t, const uint8_t *, const unsigned, uint8_t *);
 unsigned pgp_is_hash_alg_supported(const pgp_hash_alg_t *);
 
+#define PGP_PROTECTED_AT_SIZE 15
+
+typedef struct pgp_key_t pgp_key_t;
+
+typedef struct pgp_seckey_t pgp_seckey_t;
+
+typedef pgp_seckey_t *pgp_seckey_decrypt_t(const pgp_key_t *key, FILE *passfp);
+
 /** pgp_seckey_t
  */
-typedef struct pgp_seckey_t {
```

Comment:
Because I added to `pgp_seckey_t` field `pgp_seckey_decrypt_t *decrypt_cb` that returns `pgp_seckey_t`;